demo/desktop/vmf-player/MetadataProvider/MetadataProvider.cpp

```diff
@@ -301,10 +303,10 @@ void MetadataProvider::execute()
             vmf::Format::AttribMap attribs;
             vmf::Format::ParseCounters c;
 
-            char buf[10240];
-
             if (!m_exiting)
             {
+                char buf[bufSize];
```

Comment:
2+ concurrent xfers on same static buffer...

On 04/22/2016 11:10 AM, Andrey Pavlenko wrote:
>
> In demo/desktop/vmf-player/MetadataProvider/MetadataProvider.cpp 
> <https://github.com/apavlenko/vmf/pull/64#discussion_r60704226>:
>
> >              if (!m_exiting)
> >              {
> > +                char buf[bufSize];
>
> why not to make this buffer static?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/apavlenko/vmf/pull/64/files/bdf5cf8cc023f114d68b3eb0f69e027f9f821655..aa5c5d0514964726321eef42169f6865292a280a#r60704226>
>